src/loaders/express/tests/helmet.spec.ts

```diff
@@ -0,0 +1,139 @@
+import helmet from 'helmet'
+import { mocked } from 'ts-jest/utils'
+
+import config from 'src/config/config'
+import featureManager from 'src/config/feature-manager'
+
+import helmetMiddlewares from '../helmet'
+
+describe('helmetMiddlewares', () => {
+  afterEach(() => {
+    jest.clearAllMocks()
+  })
+
+  it('should load the value of helmet.xssFilter()', () => {
+    jest.mock('helmet')
+    const mockHelmet = mocked(helmet, true)
+    mockHelmet.xssFilter = jest.fn().mockImplementation(() => 'xss')
+    const xssValue = helmetMiddlewares()[0]
```

Comment:
I'm not sure indexing into the return value of the module is the right approach. this makes the tests highly restrictive - future devs cannot even change the order of the function calls without breaking the tests, even if the code is perfectly functional. what's more important here is:
1. The functions are all called with the correct arguments. Look at the matchers `toHaveBeenCalled` and `toHaveBeenCalledWith`.
2. The return value of the module contains the expected outputs of all the functions that have been called. You can call the `helmetMiddleWares` function with all the `helmet` functions mocked, and expect the array returned to contain all your mocked return values.

Reply:
thanks! this is useful guidance, see if the revised tests are ok